src/index.ts

```diff
@@ -1,14 +1,10 @@
 import express from 'express'
 
-import { ENVIRONMENT, PORT, VERSION, WEB3_PROVIDER_URL } from './config'
+import { ENVIRONMENT, PORT, VERSION } from './config'
 import { initDatabase } from './database/db'
 import { pollers } from './polling'
 
 export default async function initApp(): Promise<express.Application> {
-  if (!WEB3_PROVIDER_URL) {
```

Comment:
this didn't actually do anything because `WEB3_PROVIDER_URL` defaulted to the string `'UNDEFINED'`